jsondown.js

```diff
@@ -1,6 +1,33 @@
 var util = require('util');
-var fs = require('fs');
+var os = require('os');
 var MemDOWN = require('memdown');
+var runtime
+var fs
+
+// If global 'runtime' exists
+(os.hostname() === 'runtime') ?
+  (() => {
+    var runtime = require('runtimejs')
+    var fatfs = require('fatfs')
+    var disk = runtime.block.devices[0];
+    var blockInterface = {
+      sectorSize: disk.formatInfo.sectorSize,
+      numSectors: disk.formatInfo.totalSectorCount,
+      readSectors(i, dest, cb) {
+        disk.read(i, dest)
+          .then(cb.bind(null, null))
+          .catch(cb);
+        },
+        writeSectors(i, data, cb) {
+          disk.write(i, data)
+            .then(cb.bind(null, null))
+            .catch(cb);
+          }
+    }
+    fs = fatfs.createFileSystem(blockInterface);
+  })() :
+  fs = require('fs')
```

Comment:
Could this chunk of code be put in a separate file, like `runtime-fs.js` or something, maybe with a comment at the top explaining what the code is for? This way the `fs` import could just be e.g.:

```js
var fs = (os.hostname() === 'runtime') ? require('./runtime-fs') : require('fs');
```